cupyx/scipy/signal/_filter_design.py

```diff
@@ -0,0 +1,294 @@
+
+import operator
+import warnings
+
+import cupy
+from cupyx.scipy._lib._util import float_factorial
+
+EPSILON = 2e-16
+
+
+def _is_int_type(x):
```

Comment:
This function was causing issues for me in https://github.com/cupy/cupy/pull/7537, had to switch to 
https://github.com/cupy/cupy/pull/7537/files#diff-f34f42a4a1162ce3b414e4874a4577e9f19165b6171c21ba3d4139086b39fc86R10